### ISSUE TITLE
is it possible to disable tests ?

### ISSUE THREAD

**Issue by vtorri** [google/highway#410]
Hello

I've looked at CMakeLists.txt, but i haven't found any way to disable the build of tests ? Is it possible ? if not, is it possible to add an option to disable them ?

**Comment by vtorri** [comment]
@jan-wassenberg if I add `-DBUILD_TESTING:BOOL=FALSE`, here is the cmake warning : 

```
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
```

**Comment by jan-wassenberg** [comment]
Yes, that's normal. Even with BUILD_TESTING we build the hwy_test library, but we do not build/run the actual tests using this library. Does that work for you?

**Comment by jan-wassenberg** [comment]
There is indeed an install target after doing a fresh (with previous build directory deleted) "cmake ..".
Is it possible that HWY_EXAMPLES_TESTS_INSTALL=OFF from previous experimentation? That would disable the install target.
